PermissionsUtilsUI/src/main/AndroidManifest.xml

```diff
@@ -3,12 +3,23 @@
     package="com.drivequant.drivekit.permissionsutils">
 
     <uses-permission android:name="android.permission.BLUETOOTH" />
+
     <application>
+        <activity android:name=".diagnosis.activity.AppDiagnosisActivity" />
         <activity android:name=".permissions.activity.RequestPermissionActivity" />
         <activity android:name=".permissions.activity.BasePermissionActivity" />
         <activity android:name=".permissions.activity.LocationPermissionActivity" />
         <activity android:name=".permissions.activity.BackgroundTaskPermissionActivity" />
         <activity android:name=".permissions.activity.ActivityRecognitionPermissionActivity" />
-    </application>
 
+        <provider
+            android:name=".diagnosis.provider.PermissionFileProvider"
+            android:authorities="drivekit-permission-ui.provider"
```

Comment:
@mohamed-garci It might causes conflict when building in out white label project. Have a look in the AndroidManifest.xml on Vehicle UI module :

```<provider
            android:name=".utils.BaseFileProvider"
            android:grantUriPermissions="true"
            android:exported="false"
            android:authorities="${applicationId}.provider">
            <meta-data
                android:name="android.support.FILE_PROVIDER_PATHS"
                android:resource="@xml/file_provider_paths"/>
        </provider>
```

You can also build the modules in local Maven repository and try to compile in white label project

Reply:
Indeed, i have to update my branche to get these changes.

Reply:
Done